jenkins/gradle-check-assemble.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Should this file be called `gradle-check.sh`?

Reply:
I agree, this should not be gradle check assemble.